_shared/project/tox.ini

```diff
@@ -66,12 +66,16 @@ deps =
     pip-sync-faster
 {% elif cookiecutter._directory == 'pypackage' %}
     dev: ipython
+{% if cookiecutter.get("linter") == "ruff" %}
+    format,checkformatting,lint: ruff
+{% else %}
```

Comment:
Packages don't use requirements files, instead the dependencies for formatting, linting, testing, etc are listed here in `tox.ini` (and not pinned)